### PR TITLE
feat(android): Add support for upcoming React Native 0.73

### PR DESCRIPTION
**Pull request by Arjan-Zuidema**
Reference: https://github.com/react-native-community/discussions-and-proposals/issues/671

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -113,7 +113,31 @@ if (isNewArchitectureEnabled()) {
...
+            "<manifest $PACKAGE_PROP "
+        )
+    }
+}
+manifestContent.replaceAll("  ", " ")
+manifestOutFile.write(manifestContent)
+
 android {
+  if(shouldUseNameSpace){
+    namespace = "com.reactnativecommunity.webview"
```
Comment (bachhuberdesign) [inline android/build.gradle:139]: Shouldn't the namespace be com.reactnativecommunity.cameraroll?

Comment (Arjan-Zuidema) [reply android/build.gradle:139]: Already fixed.

Comment (bachhuberdesign) [reply android/build.gradle:139]: Great, thank you!

**Pull request merged**